ls1intum/Artemis
# `Assessment`: Fix correction round for more feedback requests

### Checklist
#### General
#### Client

### Motivation and Context
When using the list of more feedback requests, viewing an answered request is not possible. This is due to a wrong correction round parameter

### Description
In #5654 I fixed a bug where the wrong correction round was used for accepted complaints. This code was also executed for accepted (= answered) feedback requests, but since there is no now result for feedback requests, the correction round must not be increased.

### Steps for Testing
1) Check that the bug described in #5654 is still fixed
2) Open the List of Feedback requests (Course Management -> Course) and open an accepted request. Check that the submission and result are shown.


### Review Progress
#### Code Review
#### Manual Tests

### Test Coverage
changes are tested
